### PR TITLE
Add API to logout a token. Actually just delete it.

### LINKED ISSUE #265: Add a logout method for tokens
Currently the tokens are valid until they expire:
https://github.com/usit-gd/mreg/blob/22e21e764103a109515ea37b28182e2094a7f029/mreg/authentication.py#L21-L22

Add a logout API to force an immediate invalidation of the token.

### PR DESCRIPTION
**Pull request by oyvindkolbu**
Resolves #265.

### CODE REVIEW

File: mreg/api/urls.py [hunk 1]
```diff
@@ -3,5 +3,6 @@
 from . import views
 
 urlpatterns = [
+    path('logout/', views.Logout.as_view()),
```
Comment (oyvindkolbu) [inline mreg/api/urls.py]: "logout" or "token-logout"? -- futureproofing...

File: mreg/api/views.py [hunk 2]
```diff
@@ -20,3 +21,10 @@ def post(self, request, **kwargs):
 
             return Response({'token': token.key})
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+class Logout(APIView):
```
Comment (oyvindkolbu) [inline mreg/api/views.py]: Ditto. Logout -> TokenLogout?

**Comment by oyvindkolbu** [comment]
See questions in code.

File: mreg/api/urls.py (hunk 1 above)
Comment (ponas) [reply mreg/api/urls.py]: token-logout!

File: mreg/api/views.py (hunk 2 above)
Comment (ponas) [reply mreg/api/views.py]: TokenLogout!

**Pull request merged**